deploy/deploy.lib

```diff
@@ -601,7 +601,7 @@ install_specs() {
     mkdir -p "${WORKSPACE:-.}/stacks"
     cp "${HOME}/stack.xml" "${WORKSPACE:-.}/stacks/${what}.xml"
 
-    spack module tcl refresh -y --delete-tree --latest --upstream-modules
+    spack module tcl refresh -y --delete-tree --latest --upstream-modules $(<${HOME}/specs.txt)
 
```

Comment:
@matz-e : this constraint should be applied only on 1st day of month, right? Because we said multiple modules will be generated during month but for new month only latest will be propagated.

Reply:
@pramodk in a later stage, the generated modules will be rsynced into the final `modules/*/YYYY-MM` structure, so I think this constraint here will do what we want.

Reply:
@pramodk see here:

https://github.com/BlueBrain/spack/blob/develop/deploy/deploy.lib#L516

Reply:
Ok, understood!